src/js/fs.js

```diff
@@ -263,26 +255,25 @@ fs.readFileSync = function(path) {
   return Buffer.concat(buffers);
 };
 
+fs.writeFile = function(path, outdata, callback) {
+  var data = ensureBuffer(outdata);
```

Comment:
That's exactly what I concerned. Please find what we will get with the below code in comparison with node.
```
fs.writeFile('message.txt', 1, (err) => {
  if (err) throw err;
  console.log('The file has been saved!');
});
```